chpl/chpl-resources/src/main/resources-dev/quartz.properties

```diff
@@ -5,7 +5,7 @@ org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.PostgreSQ
 org.quartz.jobStore.class = org.quartz.impl.jdbcjobstore.JobStoreTX
 org.quartz.jobStore.dataSource = openchpl
 org.quartz.jobStore.tablePrefix = quartz.QRTZ_
-org.quartz.jobStore.isClustered = true
+org.quartz.jobStore.isClustered = false
```

Comment:
Do we need to change these? I ask because if it doesn't hurt to leave it clustered I'd prefer to do that. Two reasons: one is just in case we deploy another appserver. The other is that it matches what's on STG/PROD, so we _might_ have an opportunity to catch something weird earlier in the process

Reply:
Change made.  Not sure I 100% agree - I tend to think the config should match the environment in which it is running.  You are correct that it doesn't seem to have an adverse effect.  Not a hill worth dying on, though :)